prime-angular-frontend/src/app/shared/shared.module.ts

```diff
@@ -130,6 +130,7 @@ import { HealthAuthSiteOverviewContainerComponent } from './components/health-au
 import { EmailFormComponent } from './components/forms/email-form/email-form.component';
 import { ChangeVendorNoteComponent } from './components/dialogs/content/change-vendor-note/change-vendor-note.component';
 import { TransferHASiteComponent } from './components/dialogs/content/transfer-ha-site/transfer-ha-site.component';
+import { TrimSpaceInputDirective } from '@lib/modules/forms/trim-space.directive';
```

Comment:
Better to group with `UpperCaseInputDirective` further up

Reply:
Updated as suggested.